app/assemblers/assemble_tracks_for_select.rb

```diff
@@ -1,19 +1,12 @@
 class AssembleTracksForSelect
   include Mandate
 
-  def initialize(tracks = default_tracks)
-    @tracks = tracks
-  end
+  initialize_with tracks: ::Track.active
```

Comment:
I'm not 100% sure if this is the same or not. I have a feeling that this will execute every time, even if `tracks` is passed in, whereas a method will only execute if `tracks:` is nil. But I'm only about 50% sure, so give it a check?